modules/rosa-understanding.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// understanding-rosa/understanding-rosa.adoc
+
+
+[id="rosa-about"]
+= About {product-title}
+
+{product-title} (ROSA) is a fully-managed OpenShift service, jointly managed and supported by both Red Hat and Amazon Web Services (AWS). Having your clusters maintained by this service gives you the freedom to focus on deploying applications.
+
+This service is licensed directly from your AWS account. ROSA pricing is consumption based and is billed directly to your AWS account.
+
+You can quickly deploy OpenShift clusters directly from the AWS console and integrate with other AWS services.
+
+[id="rosa-cluster-consoles_{context}"]
+== Cluster creation consoles
+
+To create a new cluster, start from the AWS Management console using ROSA. This integrates with the new `rosa` CLI and API to provision clusters in your AWS account. The new API for cluster creation alleviates the burden of manually deploying the cluster in your existing VPC and account.
+
+[id="rosa-consumption-experience_{context}"]
+== Consumption Experience
+
+After the clusters are created, you can operate your clusters with the OpenShift web console or with the OpenShift Cluster Manager. The ROSA service also uses OpenShift APIs. These tools provide a standardized OpenShift experience to leverage your existing skills and tools knowledge.
+
+You receive OpenShift updates with new feature releases and a shared, common source for alignment with OpenShift Container Platform. ROSA supports the same versions of OpenShift as Red Hat OpenShift Dedicated and OpenShift Container Platform to achieve version consistency.
+
+[id="rosa-integration-aws_{context}"]
+== Integration with AWS services
+
+ROSA can integrate with a range of AWS compute, database, analytics, machine learning, networking, mobile, and various application services, enabling customers to benefit from the more than 170 AWS services that scale on-demand across the globe. These AWS-native services are directly accessible to quickly deploy and scale services through the same management interface.
+
+[id="rosa-responsibilities-raci_{context}"]
+== Responsibility Assignments
```

Comment:
Sentence case.

Reply:
Actually, decide to remove that "Responsibility assignments" section because the RACI will be in the Policies section that I am adding later.